setup.sh

```diff
@@ -57,6 +57,7 @@ touch discovery-provider/.env
 
 # setup service
 if [[ "$1" != "" ]]; then
+	audius-cli auto-upgrade
```

Comment:
not sure if we should have a prompt for this or not. ideally we would just run this for all nodes. maybe we should run this and if it fails continue. thoughts @cheran-senthil ?

Reply:
I think prompt defaulting to yes is good. Can discuss offline tomorrow